lib/kubeclient.rb

```diff
@@ -1,6 +1,7 @@
 require 'json'
 require 'rest-client'
 
+require 'kubeclient/google_application_default_credentials'
```

Comment:
nit: please keep this list sorted

Reply:
I meant to do that. Then I renamed the file. I've fixed that and will squash and rebase (to address merge conflicts in README).